File: CS50/sentimental-mario-more/mario.py
from cs50 import get_int
# prompt user for input
while True:
    h = get_int("Height: ")
    # break when input > than 0
    if h >= 1 and h < 9:
        break

# loop for the height
for i in range(h):
    # loop for the space before '#'
    for _ in range(1, h - i):
        print(" ", end="")
    # loop for the '#'
    for _ in range(i + 1):
        print("#", end="")
    # print for the '\n'
    print("  ", end="")
    # loop for the '#'
    for _ in range(i + 1):
        print("#", end="")
    # new line
    print()

File: CS50/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;

    do
    {
        height = get_int("Height: ");
    }
    while (height < 1);

    for (int i = 0; i < height; i++)
    {

        for (int space = 1; space < (height - i); space++)
        {
            printf(" ");
        }

        for (int j = 0; j < (i + 1); j++)
        {
            printf("#");
        }

        printf("  ");

        for (int j = 0; j < (i + 1); j++)
        {
            printf("#");
        }

        printf("\n");
    }
    return 0;
}
